vmslave-config/ansible/roles/setup-static-file-server/handlers/main.yml

```diff
@@ -1,4 +1,5 @@
 - name: restart apache2
+  become: yes
```

Comment:
```become``` means ```you will become someone else to execute this task```, usually someone is the root user.

```vmslave11 ansible_host=123.123.123.123 ansible_user=jenkins ansible_ssh_pass=**** ansible_become_pass=****```

ansible_become_user default is root.